### PR TITLE
arm64: dts: qcom: msm8953: xiaomi-vince: general enhancements

### PR DESCRIPTION
**Pull request by M0Rf30**
This also enable:
   - sound card
   - working td4310-fhplus-e7 video panel with already existent lk2nd binding
   - wcd_codec and jack-sensing)
   - accelerometer / gyroscope (st,lsm6dsl)

I also tried @JIaxyga ak09918 edits and this is what happens
```
[   80.235760] ak8975 0-000c: Conversion timeout happened
[   80.235791] ak8975 0-000c: Error in reading axis
```

```
xiaomi-vince:~$ hwtest
Category         Model                Path            Status     Value
framebuffer      msmdrmfb             /sys/class/graphics/fb0 working    
drm              -                    /sys/class/drm/card0-DSI-1 working    1080x2160
magnetometer     ak09918              /sys/bus/iio/devices/iio:device0 broken     
accelerometer    lsm6dsl_accel        /sys/bus/iio/devices/iio:device3 working    0.99, 0.65, -9.59 g
gyroscope        lsm6dsl_gyro         /sys/bus/iio/devices/iio:device2 working    -0.01, -0.02, 0.02 rad/s
input            Synaptics E700       /dev/input/event4 working    
input            xiaomi-vince Headset Jack /dev/input/event3 working    
input            pm8941_resin         /dev/input/event2 working    
input            pm8941_pwrkey        /dev/input/event1 working    
input            gpio-keys            /dev/input/event0 working  
```

here the dmesg

https://gist.github.com/M0Rf30/2ae924e93625e80239fed53e074097bb

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/pmi8950.dtsi [hunk 1]
```diff
@@ -206,10 +206,9 @@
 			qcom,current-limit-microamp = <20000>;	// qcom,fs-curr-ua
 			qcom,ovp-millivolt = <29600>;		// qcom,ovp-mv
 			qcom,switching-freq = <800>;		// qcom,switch-freq-khz
-			qcom,enabled-strings = <0 1 2 3>;	// qcom,led-strings-list
```
Comment (alikates) [inline arch/arm64/boot/dts/qcom/pmi8950.dtsi:209]: I'm not sure if this is okay, maybe other devices rely on this being set.

**Comment by alikates** [comment]
Also there are a couple things commented in the DT. Do you plan to enable them later?

File: arch/arm64/boot/dts/qcom/pmi8950.dtsi (hunk 1 above)
Comment (M0Rf30) [reply arch/arm64/boot/dts/qcom/pmi8950.dtsi:209]: a warning in the dmesg for 5.18 says clearly that it's a duplicated, you can go only for one of them

**Comment by M0Rf30** [comment]
> Also there are a couple things commented in the DT. Do you plan to enable them later?

I plan to fully enable the device, but these are also good placeholder found with i2cdetect and lot of introspection. Could be useful for others too. And I don't think dts will be mainlined in the short term

**Comment by alikates** [comment]
Alright then, merging!

**Pull request merged**